sdk/communication/communication-network-traversal/package.json

```diff
@@ -72,9 +72,10 @@
   "prettier": "@azure/eslint-plugin-azure-sdk/prettier.json",
```

Comment:
please apply the npm script changes described in https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/test-utils/recorder/MIGRATION.md#changes-to-npm-scripts